packages/react-native/React/Base/RCTBundleURLProvider.h

```diff
@@ -52,44 +54,44 @@ RCT_EXTERN void RCTBundleURLProviderAllowPackagerServerAccess(BOOL allowed);
  * The port is optional, if not specified, kRCTBundleURLProviderDefaultPort will be used
  * The scheme is also optional, if not specified, a default http protocol will be used
  */
-+ (BOOL)isPackagerRunning:(NSString *)hostPort scheme:(NSString *)scheme;
++ (BOOL)isPackagerRunning:(NSString *)hostPort scheme:(NSString *__nullable)scheme;
 
 /**
  * Returns the jsBundleURL for a given bundle entrypoint and
  * the fallback offline JS bundle if the packager is not running.
  */
-- (NSURL *)jsBundleURLForBundleRoot:(NSString *)bundleRoot fallbackURLProvider:(NSURL * (^)(void))fallbackURLProvider;
+- (NSURL *__nullable)jsBundleURLForBundleRoot:(NSString *)bundleRoot fallbackURLProvider:(NSURL * (^)(void))fallbackURLProvider;
```

Comment:
The closure can be null, so we need to annotate that as nullable (the internal linter is complaining)

Reply:
Fixed